core_lib_generator/template_core_lib/MANIFEST.in

```diff
@@ -5,5 +5,5 @@ include README.md
 
 global-exclude *.pyc
 global-exclude __pycache__
-recursive-include hydra_plugins/* *.yaml
-recursive-include core_lib_template/config/* *.yaml
+recursive-include template_core_lib/hydra_plugins/* *.yaml

```

Comment:
please run `python setup.py sdist bdist_wheel` from command line and make sure all YAML files are in the output generated file (`dist` folder)

Reply:
![image](https://user-images.githubusercontent.com/98592125/159661326-a55e9af1-de64-4f31-b955-140c331be79f.png)

getting the following error

Reply:
show me tommorow after you finish with all comments